Partition (WBP)/Functions/VI0049C.VTI0049.fun.yml

```diff
@@ -140,6 +141,48 @@
     
     Endroutine
     
+    *
+    * Execute SQL in LANSA and log the time
+    *
+    Mthroutine Name(mthExecuteSqlInLansa)
+    Define_Map For(*BOTH) Class(#BIFA002A) Name(#this_rc)
+    Define Field(#Start) Type(*DEC) Length(10) Decimals(3)
+    Define Field(#Finish) Type(*DEC) Length(10) Decimals(3)
+    Define Field(#Duration) Type(*DEC) Length(7) Decimals(0)
+    Define Field(#DB_Type) Type(*CHAR) Length(32)
+    Define Field(#LANSA) Type(*CHAR) Length(32)
+    Define Field(#LansaTime) Type(*CHAR) Length(32)
+    Def_List Name(#w_lLansaExecutionList) Fields(#DB_Type #LANSA #LansaTime) Type(*WORKING) Entrys(*MAX)
+    
+    #LF_rc := OK
+    #LANSA := ' ' + 'LANSA'
```

Comment:
Why is the space required?

Reply:
The Logfile was written this way for LANSA timings without space when compared to Literal and Bind timings which had spaces :                                                                                      
MSSQLS,LANSA,234
MSSQLS, Literal, 137
MSSQLS, Bind, 36

Reply:
> Why is the space required?

I have removed the spaces and committed. Kindly review the log files and let me know your comments. Thank you

Reply:
Please put the space back. I didn't ask you to remove it. It was just a question.

The Table is misnamed. PerformanceMetricsTable.txt

The rest of it looks to be good.

Reply:
Fixed. Attaching sample table files. 
[PerformanceMetricsTable.txt](https://github.com/lansa/db-regression/files/14107370/PerformanceMetricsTable.txt)

Reply:
My comment above was in relation to PerformanceMetricsLog.txt. **The LANSA entry needs the space added back.** That was the code that I highlighted.

You have now added a space to PerformanceMetricsTable.txt. It works less well with Excel. **Please remove that space.**